interceptor_test.go

```diff
@@ -1687,21 +1600,21 @@ func TestStreamingClientTracing(t *testing.T) {
 	require.Equal(t, spanRecorder.Ended()[0].Status().Code, codes.Unset)
 	assertSpans(t, []wantSpans{
 		{
-			spanName: pingv1connect.PingServiceName + "/" + CumSumMethod,
+			spanName: pingv1connect.PingServiceName + "/" + PingStreamMethod,
```

Comment:
For these values, we could also use the generated procedure name constants with a utility function to trim the leading slash. Seems semantically a little clearer to me - the span name is the full procedure name.

This is fine too - just offering up an option in case you didn't consider it.

Reply:
Have to remove the leading slash, want `<service>/<method>`.